+lumofile/write_NIRS.m

```diff
@@ -276,12 +276,10 @@
   end
   
 else
-  
-  
+    
   % No events recorded
-  s = [];
-  CondNames = {};
-  
+  s = zeros([size(t, 1) 1]); % Add a zeros stimulus matrix for compatibility with Homer2
+  CondNames = {'empty'}; % Add element to the cell for compatibility with Homer2
```

Comment:
Why not `CondNames = {''}`?

Reply:
Question still stands?

Reply:
I added 'empty' so that it would be clearer that no labels for event markers are present. Of course, both works as long as the length of the cell array matches the columns in 's'